numpy/core/tests/test_ufunc.py

```diff
@@ -1859,7 +1859,7 @@ def test_invalid_args(self):
         # gh-7961
         exc = pytest.raises(TypeError, np.sqrt, None)
         # minimally check the exception text
-        assert 'loop of ufunc does not support' in str(exc)
+        assert 'loop of ufunc does not support' in exc.value.args[0]
```

Comment:
Lets see if this work. It does work locally on older pytest, although it is not the prettiest code...

Reply:
can you use `assert exc.match('loop of ufunc does not support')`?

Reply:
Or perhaps `... in str(exc.value)` if you're looking for a minimal change

Reply:
Thanks, thought there must be a better way. It works locally, lets see what Travis thinks...